src/util.ts

```diff
@@ -35,9 +35,12 @@ export function nop() {
  * @param rootDir Directory where the tsconfig.json should be found.
  */
 export async function getTSConfig(
-    rootDir: string, parmReadFilep?: ReadFileP): Promise<any> {
+    rootDir: string, paramReadFilep?: ReadFileP): Promise<any> {
   const tsconfigPath = path.join(rootDir, 'tsconfig.json');
-  const json = await(parmReadFilep || readFilep)(tsconfigPath, 'utf8');
+  if (!paramReadFilep) {
```

Comment:
nit: more idiomatic to do `paramReadFilep = paramReadfilep || readFilep`. Can we rename the variable because it no longer contains just the parameter value though.

Reply:
I named it `customReadFilep`.